services/question/README.md

```diff
@@ -0,0 +1,428 @@
+# Question Service User Guide
+
+## Pre-requisites
+
+You need to create the `.env` files from the provided `.env.sample` files in both the root directory and
+the `services/question` directory.
+
+1. Open the Command Prompt.
+2. Navigate to the root directory of the project.
+3. Run the following command to create the `.env` files:
+```cmd
+cp .env.sample .env
+cp services/question/.env.sample services/question/.env
+```
+4. After setting up the .env files, build the Docker images and start the containers using the following command:
+```cmd
+docker compose build
+docker compose up -d
+```
+5. To stop and remove the containers and associated volumes, use the following command:
+```cmd
+docker compose down -v
+```
+
+## Get Questions
+
+This endpoint allows the retrieval of all the questions in the database. If filter by (optional) parameters, questions
+that matches with parameters will be returned; if no parameters are provided, all questions will be returned.
+
+- **HTTP Method**: `GET`
+- **Endpoint**: `/questions`
+
+### Parameters:
+
+- `title` (Optional) - Filter by question title.
+- `description` (Optional) - Filter by question description.
+- `topics` (Optional) - Filter by topics associated with the questions.
```

Comment:
I think it would make more sense for them to pass in an array of topics? Or if you prefer the comma separated method, could you specify it in the README?

Reply:
Ah okay, I understand what you mean. Okay, I will make the necessary changes.